common/data_refinery_common/migrations/0019_auto_20190404_1427.py

```diff
@@ -0,0 +1,36 @@
+# Generated by Django 2.1.5 on 2019-04-04 14:27
+
+from django.db import migrations
+from data_refinery_common.utils import get_env_variable
+
+
+# We want this to throw if it can't access this, no point in running a
+# migration to set everything to a bad value.
+S3_BUCKET_NAME = get_env_variable("S3_QN_TARGET_BUCKET_NAME")
+
+
+def update_qn_bucket(apps, schema_editor):
+    '''Sets the s3_bucket for QN Targets to a bucket just for them.
+
+    Based off of:
+    https://simpleisbetterthancomplex.com/tutorial/2017/09/26/how-to-create-django-data-migrations.html
+
+    We can't import the ComputedFile model directly as it may be a newer
+    version than this migration expects. We use the historical version.
+    '''
+    # Pagination isn't necessary here because we have very few QN targets.
+    ComputedFile = apps.get_model('data_refinery_common', 'ComputedFile')
+    for computed_file in ComputedFile.objects.filter(is_qn_target=True):
+        computed_file.s3_bucket = S3_BUCKET_NAME
```

Comment:
Why does this not automatically move the result from one bucket to the other? How will this process occur?

Reply:
Yeah this is a good point. Looking back at this now I also think I should have included code to move the data too. Not sure why I didn't before, but I'll add that in.